tools/generators/migrate-converged-pkg/index.ts

```diff
@@ -752,7 +752,19 @@ function updateLocalJestConfig(tree: Tree, options: NormalizedSchema) {
 
   tree.write(options.paths.jestConfig, templates.jest(config));
 
-  if (!tree.exists(jestSetupFilePath)) {
+  if (tree.exists(jestSetupFilePath)) {
```

Comment:
based on your findings, we need to make this check more bulletproof - meaning to add check if particular package contains `@fluentui/a11y-testing` as devDep

Reply:
Or just convert a11y-testing to use testing-library--turns out it was pretty easy. #22262

Reply:
yuuup